packages/server/src/adapters/node-http/types.ts

```diff
@@ -64,6 +64,7 @@ export type NodeHTTPHandlerOptions<
    */
   middleware?: ConnectMiddleware;
   maxBodySize?: number;
+  preparsedBody?: boolean;
```

Comment:
My gut feeling is we don't want to expose this on the public API, it would be an internal flag which each adapter can set. If there were a way to configure Next to _not_ parse JSON before passing it on, we would probably have documented that as part of setup I think, and most other adapters are lower level anyway

Reply:
@Nick-Lucas There's a way to configure Next not to parse, I mentioned that in the issue, also I added it to the docs in this PR. With body-parser disabled this flag is not required, it works properly.

Regarding flag being internal, I agree, but I am not that familiar with the src, so I was unsure where is the best place to put it, if you can suggest, I will update the PR

Reply:
https://nextjs.org/docs/api-routes/request-helpers

Right sorry I had the understanding from before that Nextjs didn't allow configuration of this behaviour. 

So don't you think the functionality work here is redundant? Rather than saying "if you're not using batching and you have bodyParser enable then set this option on tRPC" our docs should just be saying "when setting up tRPC you should disable bodyParser"?

Conditionals add complexity, and we're all about simplicity where we can be.